impl/src/test/java/io/kroxylicious/testing/kafka/KafkaClusterTest.java

```diff
@@ -68,6 +70,12 @@
 @Timeout(value = 2, unit = TimeUnit.MINUTES)
 class KafkaClusterTest {
 
+    private static final System.Logger LOGGER = System.getLogger(KafkaClusterTest.class.getName());
+
+    static {
+        LOGGER.log(System.Logger.Level.INFO, "Kafka on classpath is version: {0}", AppInfoParser.getVersion());
```

Comment:
I just want to make it easy to see that the test matrix is doing the right thing.